BranchMetrics/react-native-branch-deep-linking-attribution
# SDK-1666 prep 5.7.0 release

## Reference
SDK-1666 Prep 5.7.0 Release

## Summary
Update Android SDK to 5.2.7
Update iOS SDK to 1.45.2
Fix ~creation_source type
Add export for success and error event

## Motivation
Update underlying native SDKs

## Type Of Change

## Testing Instructions
The main things to verify are the payloads contain the correct SDK and plugin versions.
Also verify that the dependabot merges didn't break anything.










cc @BranchMetrics/saas-sdk-devs for visibility.
